src/pages/index.html

```diff
@@ -1,6 +1,330 @@
 {% extends "layout.html" %}
 
+
+{% import 'component/cardServices.html' as card %}
+{% import 'component/cardprogress.html' as card2 %}
+{% import 'component/cardportofolio.html' as card3 %}
+
 {% block content %}
-<h1>master</h1>
+
+
+
+<section id="header-sec" class="header">
+    <div class="mini-header">
+
+        <div class="container ">
+            <div class="row  align-items-lg-center">
+                <div class="col-lg-5 ">
+                    <div class="info text-center text-lg-start">
+                       <h4>Hi I am</h4> 
+                       <h3>Mahmood Fazile</h3>
+                       <h2>UI/UX designer</h2>
+                       <div class="social justify-content-center justify-content-lg-start">
+                            <div>
+                                <a href="#">    
+                                  <img src="assets/images/social1.svg" alt="insta">
+                                </a>
+                            </div>
+                            <div>
+                                <a href="#">    
+                                    <img src="assets/images/social2.svg" alt="linkedin">
+                                  </a>
+                            </div>
+                            <div>
+                                <a href="#">    
+                                    <img src="assets/images/social3.svg" alt="web">
+                                  </a>
+                            </div>
+                            <div>
+                                <a href="#">    
+                                    <img src="assets/images/social4.svg" alt="be">
+                                  </a>
+                            </div>
+                           
+                       </div>
+                       <div class="btn justify-content-center justify-content-lg-start">
+                            <button class="btn1">Hire Me</button>
+                            <button class="btn2">Downlead CV </button>
+                       </div>
+                       
+                       <div class="experiences text-start">
+                            <div class="exp-1">
+                                <h4><span>5+</span>Experiences</h4>
```

Comment:
This is not a title it should be a <p> or <span>

Reply:
done

Reply:
smae issue like social icons  anchor inside div